adaptive/learner/triangulation.py

```diff
@@ -287,23 +287,28 @@ def __init__(self, coords):
             raise ValueError("Please provide at least one simplex")
 
         coords = list(map(tuple, coords))
-        vectors = np.subtract(coords[1:], coords[0])
-        if np.linalg.matrix_rank(vectors) < dim:
-            raise ValueError(
-                "Initial simplex has zero volumes "
-                "(the points are linearly dependent)"
-            )
+        if _check_vertices:
+            vectors = np.subtract(coords[1:], coords[0])
+            if np.linalg.matrix_rank(vectors) < dim:
+                raise ValueError(
+                    "Initial simplex has zero volumes "
```

Comment:
It's not necessarily a single simplex at this point; this should read "Hull has a zero volume", right?

Reply:
Yes. This error message was already wrong before I touched it. Maybe I should update it